1.7/usage/managing-services/creating.md

```diff
@@ -0,0 +1,222 @@
+    
+    -  `config.json` -  configuration properties supported by the package, represented as a json-schema.
+    -  `package.json` -  high-level metadata about the package.
+    -  `marathon.json.mustache` - a [mustache](http://mustache.github.io/) template that when rendered will create a [Marathon](http://github.com/mesosphere/marathon) app definition capable of running your service.
+    -  `resource.json` - contains all of the externally hosted resources (e.g. Docker images, HTTP objects and images) that are required to install the application.
+
+1.  Install the Universe repository pre-commit hooks.
+
+    ```bash
+    $ bash scripts/install-git-hooks.sh
+    ```
+
+1.  Create a new directory for the your package in `repo/packages/`. For example, to create a package named `helloworld` the structure is `repo/packages/H/helloworld/0`. The `0` directory corresponds to the release number of the package.  If you look at other packages in Universe, many have multiple releases (directories `0`, `1`, `2`, etc).
+
+Now, you're ready to start creating your package!  
+
+## Step One: Create package.json
+
+Every package in Universe must have a package.json file which specifies the high level metadata about the package. This file specifies the highest-level metadata about the package (comparable to a `package.json` in Node.js or `setup.py` in Python).
+
+Currently, a package can specify one of two values for `.packagingVersion`, either 2.0 or 3.0. The version is declared will dictate which other files are required for the complete package as well as the schemas all the files must adhere to. 
+
+Below is a snippet that represents a version 2.0 package. See [`repo/meta/schema/package-schema.json`](https://github.com/mesosphere/universe/blob/version-3.x/repo/meta/schema/package-schema.json) for the full JSON schema outlining what properties are available for each corresponding version of a package.
+
+```javascript
+{
+  "packagingVersion": "2.0", // use either 2.0 or 3.0
+  "name": "foo", // your package name
+  "version": "1.2.3", // the version of the container that you are launching
+  "tags": ["mesosphere", "framework"],
+  "maintainer": "help@bar.io", // who to contact for help
+  "description": "Does baz.", // description of package
+  "scm": "https://github.com/bar/foo.git", 
+  "website": "http://bar.io/foo", 
+  "framework": true,
+  "postInstallNotes": "Have fun foo-ing and baz-ing!"
+}
+```
+
+## Step Two: Create resource.json
+
+This file declares all the externally hosted assets the package will need &mdash; for example: Docker containers, images, or native binary CLI.  See the resource schema version [2.0](https://github.com/mesosphere/universe/blob/version-3.x/repo/meta/schema/v2-resource-schema.json) or [3.0](https://github.com/mesosphere/universe/blob/version-3.x/repo/meta/schema/v3-resource-schema.json) for the available assets for each.
```

Comment:
Let's replace the links with a link to the readme: https://github.com/mesosphere/universe#resourcejson